tests/tck/features/MatchAcceptance2.feature

```diff
@@ -1387,6 +1387,7 @@ Feature: MatchAcceptance2
       | (:Artist:B) | (:Artist:C) |
     And no side effects
 
+@skip
```

Comment:
This test had accidentally passed previously. On master, it produces the erroneous plan (note line 11):
```
 1) "Results"
 2) "    Distinct"
 3) "        Project"
 4) "            OR Apply Multiplexer"
 5) "                Cartesian Product"
 6) "                    All Node Scan | (a)"
 7) "                    All Node Scan | (b)"
 8) "                Semi Apply"
 9) "                    Argument"
10) "                    Conditional Variable Length Traverse (Expand Into) | (b:MissingLabel)<-[anon_1:T*1..INF]-(a)"
11) "                        Conditional Traverse | (b:MissingLabel)->(b:MissingLabel)"
12) "                            Argument"
13) "                AND Apply Multiplexer"
14) "                    Argument"
15) "                    Filter"
16) "                        Argument"
17) "                    Semi Apply"
18) "                        Argument"
19) "                        Expand Into | (a)->(b:TheLabel)"
20) "                            Argument"
```